AutoCollection/WebSnippet/fetchSnippet.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const fs = require('fs');
+const https = require('https');
+
+let snippetUrl = 'https://raw.githubusercontent.com/microsoft/ApplicationInsights-JS/master/AISKU/snippet/snippet.js';
+let snippetMinUrl = 'https://raw.githubusercontent.com/microsoft/ApplicationInsights-JS/master/AISKU/snippet/snippet.min.js';
```

Comment:
nice! - Although this does mean that anything checked in -- even if not released will break this...

Perhaps we should consider publishing the snippet to the CDN as well and then change this to fetch from there.